portal_pages/templates/portal_pages/case_study_page_add.html

```diff
@@ -0,0 +1,206 @@
+{% extends "base.html" %}
+{% load static %}
+{% load wagtailcore_tags wagtailroutablepage_tags %}
+
+{% block extra-header %}
+    <script type="text/javascript" src="{% static 'js/tinymce/tinymce.min.js' %}"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+    <script type="text/javascript" src="{% static 'js/chosen/chosen.jquery.min.js' %}"></script>
+{% endblock %}
+
+{% block extra-css %}
+    <link href="{% static 'css/chosen.css' %}" rel="stylesheet"/>
+{% endblock %}
+
+{% block content %}
+<div class="page-width float-layout"><div class="cushion">
+
+    <form enctype="multipart/form-data" action="." method="post" onsubmit="return checkFormAndSubmit();">
+    {% csrf_token %}
+
+    <div class="two-thirds"><div class="cushion">
+
+        <div class="cushion">
+            {{ case_study_index.submit_info|richtext }}
+        </div>
+
+        <div class="cushion">
+            {% if form.errors or document_form.errors %}
+                <p class="cushion errorlabel">Please correct the errors below.</p>
+            {% endif %}
+
+            {{ form.honeypot }}
+
+            {{ form.media }}
```

Comment:
This is not a field but javascript...should be rendered as part of the extra-js block and there are no errors associated with it...(below line should be removed I think unless I'm missing something...)